lib/suma/async/upsert_frontapp_contact.rb

```diff
@@ -5,7 +5,7 @@
 class Suma::Async::UpsertFrontappContact
   extend Amigo::Job
 
-  on(/suma\.member\.+(created|updated)$/)
+  on(/suma\.member+(\.created|\.updated)$/)
```

Comment:
I think the right regex here is `/^suma\.member\.(created|updated)$/`. But the old one wouldn't match `suma.member.session.created`: https://rubular.com/r/xgVsdW8SL0LasS